packages/roosterjs-content-model-dom/lib/domUtils/hiddenProperties/hiddenProperty.ts

```diff
@@ -2,10 +2,18 @@
  * @internal
  */
 export interface HiddenProperty {
+    /**
+     * A marker string that can be used to identify a specific paragraph in the DOM.
+     * This is useful for scenarios where you need to track or manipulate specific paragraphs
+     */
+    paragraphMarker?: string;
+
     /**
      * Specify we should not delete this element when delete/backspace key is pressed
      */
     undeletable?: boolean;
+
+    // Add more properties as needed
```

Comment:
Is this comment needed?

Reply:
it is optional, just let others know they key keep adding things here